build.gradle

```diff
@@ -175,18 +167,33 @@ tasks.register('integrationTest', Test) {
 }
 
 dependencies {
-    api platform("com.exactpro.th2:bom:4.2.0")
-    api 'com.exactpro.th2:grpc-common:4.2.0-dev'
-    api "com.exactpro.th2:cradle-core:${cradleVersion}"
-    api 'io.github.microutils:kotlin-logging:2.1.21'
+    api platform("com.exactpro.th2:bom:4.4.0-update-libs-5335052162-8affa4a-SNAPSHOT")
+    api('com.exactpro.th2:grpc-common:4.3.0-TH2-4887-vulnerabilities-5346500826-f180a19-SNAPSHOT') {
+        because('protobuf transport is main now, this dependnecy should be moved to grpc, mq protobuf modules after splitting')
+    }
+    api("com.exactpro.th2:cradle-core:${cradleVersion}") {
+        because('cradle is included into common library now, this dependnecy should be moved to a cradle module after splitting')
+    }
+    api('io.netty:netty-buffer') {
+        because('th2 transport protocol is included into common library now, this dependnecy should be moved to a th2 transport module after splitting')
+    }
 
     jmh 'org.openjdk.jmh:jmh-core:0.9'
     jmh 'org.openjdk.jmh:jmh-generator-annprocess:0.9'
 
     implementation 'com.google.protobuf:protobuf-java-util'
-    implementation 'com.exactpro.th2:grpc-service-generator:3.2.2'
+    implementation 'com.exactpro.th2:grpc-service-generator:3.4.0-TH2-4887-vulnerabilities-5346040251-1bed6cd-SNAPSHOT'
     implementation "com.exactpro.th2:cradle-cassandra:${cradleVersion}"
 
+    def autoValueVersion = '1.10.1'
+    implementation "com.google.auto.value:auto-value-annotations:$autoValueVersion"
+    kapt("com.google.auto.value:auto-value:$autoValueVersion") {
+        //FIXME: Updated library because it is fat jar
+        // auto-value-1.10.1.jar/META-INF/maven/com.google.guava/guava/pom.xml (pkg:maven/com.google.guava/guava@31.1-jre, cpe:2.3:a:google:guava:31.1:*:*:*:*:*:*:*) : CVE-2023-2976, CVE-2020-8908
```

Comment:
It is preprocessing. This library does not go into the final class path. We should configure the dependencyCheck correctly and exclude kapt configurations from checking

Reply:
I think we should check all by default to know about all vulnerabilities